documentation/projects/proposals/additional_search_views/20230719-implementation_plan_additional_search_views.md

```diff
@@ -0,0 +1,263 @@
+2023-07-20 Implementation Plan: Additional Search Views
+
+**Author**: @obulat
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] TBD
+- [ ] TBD
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/410)
+- [Project Proposal](https://docs.openverse.org/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.html)
+
+## Overview
+
+<!-- An overview of the implementation plan, if necessary. Save any specific steps for the section(s) below. -->
+
+This plan describes the changes that need to be added on the frontend: the new
+components, pages and store.
+
+## Expected Outcomes
+
+<!-- List any succinct expected products from this implementation plan. -->
+
+Three collection pages are added to Openverse:
+
+- items with the selected tag
+- items from the selected source (provider)
+- items by the selected creator. The pages re-use the existing image grid and
+  audio collection views and can load more images or audio items on the Load
+  more button click. The single result pages are updated to add the links to
+  these pages, as seen in the Figma mockups: [link to Figma mockup]
+
+## Outlined Steps
+
+<!-- Describe the implementation step necessary for completion. -->
+
+### Page changes
+
+#### Create a page for `tag` / `creator` /`source` collections.
+
+Nuxt allows creating nested dynamic routes like
+`/pages/_collection/_mediaType/_term`.
+
+Here, the collection can be `tag`, `creator` or `source`. `mediaType` can be one
+of the supported media types (currently, `image` and `audio`). `term` refers to
+the actual value of the tag, creator or source name.
```

Comment:
That's fantastic! It could be useful for SEO as well. Speaking of which, what considerations do we need to apply to `robots.txt` for this change, if any? I don't really understand our current approach to robots.txt, is it leftover from when we were iframed? @zackkrida I believe you may know.

Reply:
Our current robots.txt is blank, which allows all pages of the site to be crawled and indexed. That works exactly the same as:

```
User-agent: *
Disallow:
```


The staging site is blocked from indexing. 

I don't think we need to do anything here. These pages can be crawled and indexed.

Reply:
These new pages are essentially search views with static URLs though (not query params). Should that kind of thing be indexed?

Sorry if this is already discussed somewhere. Might be helpful to document the rationale of our robots.txt in the docs site.